<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="object">
    <link name="link">
      <collision name="collision">
        <geometry>
          <cylinder>
            <radius>1</radius>
            <length>1.2</length>
          </cylinder>
        </geometry>
      </collision>
      <visual name="visual">
        <geometry>
          <cylinder>
            <radius>1</radius>
            <length>1.2</length>
          </cylinder>
        </geometry>
      </visual>
    </link>
    <plugin name="gazebo_plugin" filename="libgazebo_object_gazebo_plugin.so" />
  </model>
</sdf>
